### ISSUE TITLE
Open a File List

### ISSUE THREAD

**Issue by jojje70** [deepskystacker/DSS#178]
Version 5.1 and 5.1.1

If you open an already saved filelist, the program cannot load some files if the file filepath contains special characters such as åäö. No error message, just absence of the affected files.

DSS-filelist that fails to load Darks and Flats:
DSS file list
CHECKED	TYPE	FILE
0	light	H_Alpha/IC_1805_Light_005.fits
1	light	OIII/IC_1805_Light_001.fits
1	light	OIII/IC_1805_Light_002.fits
1	light	OIII/IC_1805_Light_003.fits
1	light	OIII/IC_1805_Light_004.fits
1	light	OIII/IC_1805_Light_005.fits
1	light	OIII/IC_1805_Light_006.fits
1	light	OIII/IC_1805_Light_007.fits
1	light	OIII/IC_1805_Light_008.fits
1	light	OIII/IC_1805_Light_009.fits
1	light	OIII/IC_1805_Light_010.fits
1	light	OIII/IC_1805_Light_011.fits
1	light	OIII/IC_1805_Light_012.fits
1	light	OIII/IC_1805_Light_013.fits
1	light	OIII/IC_1805_Light_014.fits
1	light	OIII/IC_1805_Light_015.fits
1	dark	../../Säsong 3 2022/Kallibrering 2301/SBIG/Dark/MasterDark_ISO0_300s.tif
1	flat	../../Säsong 3 2022/Kallibrering 2301/SBIG/Flat/OIII/MasterFlat_ISO0.tif

DSS-filelist that manage to load Darks:
DSS file list
CHECKED	TYPE	FILE
0	light	H_Alpha/IC_1805_Light_005.fits
1	light	OIII/IC_1805_Light_001.fits
1	light	OIII/IC_1805_Light_002.fits
1	light	OIII/IC_1805_Light_003.fits
1	light	OIII/IC_1805_Light_004.fits
1	light	OIII/IC_1805_Light_005.fits
1	light	OIII/IC_1805_Light_006.fits
1	light	OIII/IC_1805_Light_007.fits
1	light	OIII/IC_1805_Light_008.fits
1	light	OIII/IC_1805_Light_009.fits
1	light	OIII/IC_1805_Light_010.fits
1	light	OIII/IC_1805_Light_011.fits
1	light	OIII/IC_1805_Light_012.fits
1	light	OIII/IC_1805_Light_013.fits
1	light	OIII/IC_1805_Light_014.fits
1	light	OIII/IC_1805_Light_015.fits
1	dark	../../Season 4 2023/Kallibrering/SBIG/Darks/MasterDark_ISO0_300s.tif

**Comment by perdrix52** [comment]
To remind folks the normal problem reporting path is to use the DeepSkyStacker mailing list at groups.io ...

This behaviour doesn't surprise me - it has always been the case the if DSS couldn't open the file, it just failed silently.

I will take a look to see how easy/hard this is to resolve.

**Comment by perdrix52** [comment]
I've closed this as I have had no response

**Comment by jojje70** [comment]
> 

Yes, the filelist was written with 5.1 and reopened in both 5.1 and 5.1.1. Good thing you found the cause of the problem, and I have just joined the mailing list.
